File: 160715/file/ruby/six.rb
# 6. 实现一个方法，传入一个文件名filename，返回其大小，不存在则返回空(null/nil)
file_name = 'text'

def back_file_size(file_name)
  File::exists?("#{file_name}.txt") ? file_size = File.size?("#{file_name}.txt") : file_size = nil
end

result = back_file_size(file_name)
p result

File: 160715/file/java/six.java
// 6. 实现一个方法，传入一个文件名filename，返回其大小，不存在则返回空(null/nil)
import java.io.File;

public class six{
  public static void main(String[] args) {
    String filename = "test";
    long result = judgeFileSize(filename);
    if (result == 0) {
      System.out.println("null");
    }else{
      System.out.println(result);
    }
  }

  public static long judgeFileSize(String filename){
    File file = new File(filename + ".txt");
    if (file.exists() || file.isFile()) {
      return file.length();
    }else{
      return 0;
    }
  }
}